operatorcourier/cli.py

```diff
@@ -2,11 +2,17 @@
 import sys
 from operatorcourier import api
 
+sys.tracebacklimit = 0  # this suppresses stacktrace of the CLI tool in the command line
+
+
 def main():
     """Generate the CLI bits
     """
-    parser = _CliParser()
-    parser.parse()
+    try:
+        parser = _CliParser()
+        parser.parse()
+    except Exception:  # Exception raised to CLI should not be thrown to users,
+        exit(1)        # it should just be captured by logs
```

Comment:
I'm not avare that argparser is giving traceback for user errors, only for developer errors.

This case just hides unexpected errors, you at least needs
```
except Exception as e:
   sys.exit(str(e))
```